pkg/host/virt.go

```diff
@@ -34,6 +35,22 @@ type VirtualizationEnvironment struct {
 	IsKVM bool `json:"is_kvm"`
 }
 
+var currentVirtEnv VirtualizationEnvironment
+
+func init() {
```

Comment:
oh.. there is a package named host here.. we can move the files in os package to here.